src/content/en/2021/media.md

```diff
@@ -1,20 +1,875 @@
+  content="6%",
+  classes="big-number",
+  sheets_gid="620965569",
+  sql_file="picture_distribution.sql"
+)
+}}
+
+`<picture>` is used much less frequently than `srcset`, reflecting its somewhat-niche pair of use-cases. Here's how usage breaks down between those two use cases:
+
+{{ figure_markup(
+  image="picture-feature-usage.png",
+  caption="`<picture>` feature usage.",
+  description="A bar chart showing the percent of pages which use the `media` and `type` attributes on `source` elements, in conjunction with the `picture` element. `media` is used with 48% of `picture` elements on mobile, and just over 44% on desktop. `type` is used with 37% of `picture` elements on mobile, and close to 39% of `picture` elements on desktop.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQM9deg869BD9knNdVhFNbFnUdVXeyuwzUEIgSW-2XgOBEbALtVnoFapQ5JsDxzzepj6mVoepKBmN_m/pubchart?oid=1613173002&format=interactive",
+  sheets_gid="2031063502",
+  sql_file="picture_switching.sql"
+  )
+}}
+
+Art direction appears a bit more popular than type-switching. Notably, these two features are not mutually exclusive, and, put together, the usage percentages here do not add up to 100%. This suggests that at least 15% of `<picture>` elements do not leverage either of these attributes, making those `<picture>`s  functionally equivalent to a `<span>`.
+
+### Layout
+
+Once you've embedded an image on a page, you must lay it out amongst the rest of the page's contents. There are many, many ways to do this, but we can derive a few insights about how it's generally done by zooming out and answering a couple of big-picture questions.
+
+#### Intrinsic vs extrinsic sizing
+
+As [replaced elements](https://developer.mozilla.org/en-US/docs/Web/CSS/Replaced_element), images have a natural, ["intrinsic" size](https://developer.mozilla.org/en-US/docs/Glossary/Intrinsic_Size). This is the size that they will render at by default, if there are no CSS rules placing "extrinsic" layout constraints upon them.
+
+How many images are extrinsically vs extrinsically sized?
+
+{{ figure_markup(
+  image="intrinsic-and-extrinsic-image-sizing.png",
+  caption="Intrinsic and extrinsic image sizing.",
+  description="A stacked bar chart showing the percentage of images whose width and height are determined based on extrinsic vs extrinsic sizing. There is a (for now) mysterious third category: both. The distributions between intrinsic, extrinsic, and both are quite different for height and width. For height, 60% of images are intrinsically sized, and 30% extrinsic; the remainder fall under both. For width, just 10% of images are intrinsically sized, and 66% are extrinsic; the remaining 24% are both.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQM9deg869BD9knNdVhFNbFnUdVXeyuwzUEIgSW-2XgOBEbALtVnoFapQ5JsDxzzepj6mVoepKBmN_m/pubchart?oid=769042260&format=interactive",
+  sheets_gid="576119731",
+  sql_file="image_sizing_extrinsic_intrinsic.sql"
+  )
+}}
```

Comment:
Again maybe don't use red unless you mean bad?

Reply:
Ok!

Reply:
Done